cirq-google/cirq_google/experimental/noise_models/calibration_to_noise_properties.py

```diff
@@ -0,0 +1,88 @@
+import cirq_google
+import numpy as np
+from cirq.devices.noise_properties import NoiseProperties
+
+
+def noise_properties_from_calibration(
+    calibration: cirq_google.Calibration, validate: bool = True, tolerance: float = 0.01
+):
+    """Translates between a Calibration object and a NoiseProperties object1
+    The NoiseProperties object can then be used to create a NoiseModel for a simulator.
+
+    Args:
+        calibration: a Calibration object with hardware metrics
+    """
+
+    def unpack_from_calibration(metric_name):
```

Comment:
Having to define so many methods inside another method is a red flag - can they be moved outside (or called from NoiseProperties)?